### DISCUSSION TITLE
UE 5 Support

### DISCUSSION THREAD

**Discussion by Nogbit** [AndreaCatania/RuntimeSkeletalMeshGenerator#10] [Ideas]
Any plans to add support for UE5, what are the current challenges to do so?

**Comment by vmol621** [comment]
I was able to get it working on my UE5 project with a few changes.
Manly to support UE5s Large World Coordinates. So basically converting FVector to FVector3f etc.

More info here:

https://docs.unrealengine.com/5.0/en-US/large-world-coordinates-project-conversion-guidelines-in-unreal-engine-5/

**Reply by simpple28** [reply]
Can you share the ue5 version?

**Reply by tim-trimble** [reply]
[UE5_Edtis_RuntimeSkeletalMeshGenerator.zip](https://github.com/AndreaCatania/RuntimeSkeletalMeshGenerator/files/13942430/UE5_Edtis_RuntimeSkeletalMeshGenerator.zip)

Haven't been able to actually test whether the functionality of works but here are some edits that at least compile

**Comment by BucksVR** [comment]
Hello, I'm working with Bucks County Community College on using this plugin in education. We are having some trouble though. The plugin doesn't seem to be working with any version of UE5. Is there a special plugin that we need to download?
